crt_portal/crt_portal/settings.py

```diff
@@ -158,11 +156,13 @@
     STATIC_URL = f'https://{AWS_S3_CUSTOM_DOMAIN}/{AWS_LOCATION}/'
     STATICFILES_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
     DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
-
 else:
     STATIC_URL = '/static/'
 
-STATIC_ROOT = os.path.join(BASE_DIR, "static")
+STATICFILES_DIRS = ( os.path.join(BASE_DIR, 'static'), )
+ADMIN_MEDIA_PREFIX = STATIC_URL + 'admin/'
```

Comment:
@LindsayYoung It looks like `ADMIN_MEDIA_PREFIX` is deprecated, does that match your understanding? https://docs.djangoproject.com/en/dev/releases/1.4/#django-contrib-admin